Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -49,6 +50,9 @@
 
 static const wchar_t* TAG = L"CGContext";
 
+// Coordinate offset to support CGGradientDrawingOptions
+static const float s_CGGradientStartPoint = 1E-45;
```

Comment:
LOL yep

---
In reply to: [93370154](https://github.com/Microsoft/WinObjC/pull/1535#discussion_r93370154) [](ancestors = 93370154)